src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/ProgrammableAndroid.java

```diff
@@ -424,10 +424,10 @@ protected void openScriptDownloader(Player p, Block b, int page) {
         });
 
         int index = 0;
-        int categoryIndex = 45 * (page - 1);
+        int itemGroupIndex = 45 * (page - 1);
 
         for (int i = 0; i < 45; i++) {
-            int target = categoryIndex + i;
+            int target = itemGroupIndex + i;
```

Comment:
This does not refer to item groups, it's the different categories for android scripts

Reply:
ah all right